src/s3.js

```diff
@@ -13,4 +27,9 @@ module.exports = async function(cfg) {
     });
 
     return exportFns;
+
+})
```

Comment:
I think because this return is now nested w/in `chain.resolve`, we'll no longer return `exportFns` from our exports – instead, we'll return undefined.

Perhaps instead we could return a `Promise` that is resolved with `exportFns` when our credentials are available, or rejected I guess when `err` indicates something has gone wrong?